pkg/cmd/pulumi/state_rename.go

```diff
@@ -128,32 +129,71 @@ func newStateRenameCommand() *cobra.Command {
 	var yes bool
 
 	cmd := &cobra.Command{
-		Use:   "rename <resource URN> <new name>",
+		Use:   "rename [resource URN] [new name]",
 		Short: "Renames a resource from a stack's state",
 		Long: `Renames a resource from a stack's state
 
 This command renames a resource from a stack's state. The resource is specified
-by its Pulumi URN (use ` + "`pulumi stack --show-urns`" + ` to get it) and the new name of the resource.
```

Comment:
I think we can leave a hint here about --show-urns but maybe rephrased to just comment that --show-urns can be used?

Reply:
Sure